### ISSUE TITLE
Name Suggestion & Pattern Recognition Bugs

### ISSUE THREAD

**Issue by mbeganovic3** [jordangkq/COMP-580-Project#1]
- Musical Math Tiles (not that great but just made one since I do not see that you have one)

The game is still pretty buggy especially when it comes to the pattern recognition one. For example, the sounds will not play on the computer as it gives a 404 status error in the console(check picture below) and also it does not work right on my android phone. All I could see is the shuffle and practice button, I would have to switch it to the desktop site to see the tiles and actually tap on them but it looks good on making comparisons. Also, it gives you the written direction to tap the screen to start, a blind person would not see this so maybe have a voice actually say that instead. 

![image](https://user-images.githubusercontent.com/42630924/79805023-40aef500-8333-11ea-935d-730f0d6e942e.png)

**Comment by benknoble** [comment]
I vote for MusicalMath @mbeganovic3 and I were on the same thought train :)

On Safari, I couldn't hear the button sounds. The voice generation used my French voice generator, and he pronounced everything with a French accent (making the English very difficult to understand).

I couldn't even get the shuffle to work.

**Comment by obglass** [comment]
I agree with the comments above, the comparison game worked well! However, the pattern recognition one would not let me click anything and also the shuffle and practice buttons were rather small for a visually impaired person. I like the names above too!

**Comment by shayandolikhani** [comment]
From using the program on Chrome, the comparison game works as intended! The only issue is that when you press multiple tiles in response to a prompt, it will interpret that as responses to the next few prompts, so perhaps finding a way to pause inputs so they only come in one at a time after each prompt would fix that. The pattern recognition game seems incomplete since clicking on practice, shuffle, or any of the tiles doesn't do anything.

**Comment by dkoenigs** [comment]
I like the sound of 'MusicalMath' @mbeganovic3 @benknoble .
- The pattern recognition game has some bugs. 'Practice' doesn't generate sounds when tiles are clicked.
- Shuffle mode also does not work as described

**Comment by charleshiggins** [comment]
Just wanted to quickly mirror what others have said- comparison game works great (although you can get out of sync with the voiceover if you input multiple times before the beginning of the next question). Pattern recognition game doesn't seem to be loading the audio. Interface looks great though!

**Comment by gbishop** [comment]
Still buggy but good game concept. Patterns worked for me in the first few rounds but then somehow
got out of sync and didn't play the sounds.
